docs/en/observability/tab-widgets/uptime-monitoring/run-test-suite-content.asciidoc

```diff
@@ -0,0 +1,71 @@
+// tag::heartbeat[]
+
+Copy and paste the path to your zip file into `heartbeat.yml`.
+For a full list of configuration options, see {heartbeat-ref}/monitor-browser-options.html[Browser options in the {heartbeat} documentation].
+
+[source,yml]
+----
+- name: Todos
+  id: todos
+  type: browser
+  schedule: "@every 1m"
+  params:
+    url: "https://elastic.github.io/synthetics-demo/" <1>
+  source:
+    zip_url: <2>
+      url: "https://github.com/elastic/synthetics-demo/archive/refs/heads/main.zip" <3>
+      folder: "todos/synthetics-tests" <4>
+      username: "" <5>
+      password: "" 
+----
+<1> The `params` section lets you define custom parameters to use in your script. In this example the `url` parameter is used by the test suite to determine which site to test.
```

Comment:
Perhaps to make these docs more unambiguous given `source.zip_url` also has an `url` key we could call this `target_url` or `test_url`.

Reply:
Also updated the screenshot for consistency. 🙂